lib/config.js

```diff
@@ -7,7 +7,12 @@ var autoconfig = function(overrides) {
     IP: process.env.OPENSHIFT_NODEJS_IP || '0.0.0.0',
     PORT: process.env.FH_PORT || process.env.OPENSHIFT_NODEJS_PORT || 8001,
     dataTopicPrefix: ':cloud:data',
-    persistentStore: process.env.WFM_USE_MEMORY_STORE === "true"
+    persistentStore: process.env.WFM_USE_MEMORY_STORE === "true",
+    syncOptions: {
+      "sync_frequency" : 5,
```

Comment:
That's up to the application developer. If they want a lower frequency they can change the value.

Ultimately all that stuff can probably move to environment variables for the cloud app. but it is outside the scope of this ticket.